e2e/old-arch/various-suite/prometheus-annotations.spec.ts

```diff
@@ -56,7 +56,7 @@ describe('Prometheus annotations', () => {
 
     // check for other parts of the annotations
     // min step
-    cy.get(`#${selectors.components.DataSource.Prometheus.annotations.minStep}`);
+    cy.get(`[data-testid="${selectors.components.DataSource.Prometheus.annotations.minStep}"]`);
```

Comment:
FYI - I _think_ if the selector value starts with `data-testid` then you should be able to do 

```
e2e.components.DataSource.Prometheus.annotations.minStep.title().should('exist');
```

Reply:
Good catch, thanks!